libs/service/api_service.cpp

```diff
@@ -21,47 +21,72 @@ extern "C"
 
 static ebpf_result_t
 _build_helper_id_to_address_map(
-    ebpf_handle_t program_handle, ebpf_code_buffer_t& byte_code, std::map<uint32_t, uint64_t>& helper_id_to_adddress)
+    ebpf_handle_t program_handle, ebpf_code_buffer_t& byte_code, std::vector<uint64_t>& helper_addresses)
 {
+    // Note:
+    // eBPF supports helper IDs in the range [0, MAXUINT32]
```

Comment:
actually I don't think 0 is legal either (I know the PREVAIL verifier treats 0 as illegal), so it'd be [1, MAXUINT32]

Does uBPF really support a helper ID of 0?